packages/tic-tac-toe/blueprints/component/index.js

```diff
@@ -0,0 +1,280 @@
+/* eslint-disable prefer-const */
```

Comment:
Is this a generated file or written by hand?

Reply:
This is a copy of the Ember blueprints for generating components updated to generate TS files instead of JS files. 
The current implementation of Ember CLI Typescript isn't up to date with best practices.

The solution in the community isn't clear yet so this is currently my way of iterating faster.